src/ImGui.NET/ImGuiNative.cs

```diff
@@ -239,6 +225,23 @@ public static unsafe class ImGuiNative
         [DllImport(cimguiLib)]
         public static extern float igGetItemsLineHeightWithSpacing();
 
+        // Columns
```

Comment:
Sorry about that. I reordered them to match the order in (c)imgui headers, so it should be easier to catch changes in the future.